packages/providers/src/provider.ts

```diff
@@ -648,12 +654,13 @@ export default class Provider {
    */
   async getTransactionCost(
     transactionRequestLike: TransactionRequestLike,
-    tolerance: number = 0.2
+    _tolerance: number = 0.2
```

Comment:
Why are we keeping this if it [apparently] became useless?

Reply:
@arboleya Good point. I am going to remove this param